roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1917,8 +1917,8 @@ def __init__(self, role, filename, local_facts,
             self.system_facts = ansible_facts(module, ['hardware', 'network', 'virtual', 'facter'])  # noqa: F405
             for (k, v) in self.system_facts.items():
                 self.system_facts["ansible_%s" % k.replace('-', '_')] = v
-        except UnboundLocalError:
-            # ansible-2.2
+        except (UnboundLocalError, RuntimeError):
```

Comment:
Catching ``UnboundLocalError`` and ``RuntimeError`` seems sloppy to me, but if it works it works. Is ``system_facts`` still being populated/manipulated in another thread?